Frameworks/CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -707,11 +707,14 @@ CF_INLINE uintptr_t __CFISAForTypeID(CFTypeID typeID) {
 CF_INLINE bool __CF_IsBridgedObject(CFTypeRef obj) {
     CFRuntimeBase* object = (CFRuntimeBase*)obj;
     if (!object || 
-        (object->_cfisa == 0) || 
-        (object->_cfisa == (uintptr_t)(&_OBJC_CLASS__NSCFString)) || 
+        (object->_cfisa == 0)) {
+        return false;
+    }
+
+    if ((object->_cfisa == (uintptr_t)(&_OBJC_CLASS__NSCFString)) || 

```

Comment:
I think in this case, it should be alright without one. Once we revisit CF's objc interop we may not have to worry about this.

Specifically, if everything becomes a bridged internal type, everything is Objective-C-able, and we don't have to worry about many of these checks.